### PR TITLE
[UR][L0] update L0 Loader version to v1.17.39

### PR DESCRIPTION
**Pull request by nrspruit**
-pre-commit PR for https://github.com/oneapi-src/unified-runtime/pull/2039

### CODE REVIEW

File: sycl/test-e2e/KernelFusion/queue-shortcut-functions.cpp [hunk 1]
```diff
@@ -80,7 +80,7 @@ int main() {
   // should be cancelled as a dependency with an event to be fused is found.
 
   // CHECK:      FusionStartPoint = 0:
-  // CHECK-NEXT: WARNING: Not fusing 'copy usm' command group. Can only fuse device kernel command groups.
+  // CHECK: WARNING: Not fusing 'copy usm' command group. Can only fuse device kernel command groups.
```
Comment (sommerlukas) [inline sycl/test-e2e/KernelFusion/queue-shortcut-functions.cpp:83]: Can you please provide a rationale why this change is necessary? 

Is there any additional output between the two lines?

Comment (nrspruit) [reply sycl/test-e2e/KernelFusion/queue-shortcut-functions.cpp:83]: When L0 Adapter is being used, the new L0 loader always reports the loader init in a single print. That broke this test so this needs to be changed to check for this line instead of exactly after this line.

Comment (sommerlukas) [reply sycl/test-e2e/KernelFusion/queue-shortcut-functions.cpp:83]: Ok, thanks for the explanation.

**Review by sommerlukas — APPROVED** [review]

**Review by pbalcer — APPROVED** [review]

**Comment by pbalcer** [comment]
@intel/llvm-gatekeepers please merge
